partner_event/views/event_registration_view.xml

```diff
@@ -1,31 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+<!-- © 2016 Tecnativa S.L. - Jairo Llopis
+     © 2016 Tecnativa S.L. - Vicent Cubells
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 
-<openerp>
-<data>
+<odoo>
 
-<record id="view_registration_search" model="ir.ui.view">
-    <field name="name">Search attended registrations</field>
-    <field name="model">event.registration</field>
-    <field name="inherit_id" ref="event.view_registration_search"/>
-    <field name="arch" type="xml">
-        <!-- TODO Remove if https://github.com/odoo/odoo/pull/12997 is merged -->
-        <xpath expr="//filter[@domain=&quot;[('state','=','open')]&quot;]"
-               position="attributes">
-            <attribute name="name">open</attribute>
-        </xpath>
+    <record id="view_registration_search" model="ir.ui.view">
```

Comment:
It is common in views to remove 1st indentation level because it adds no value. Those `<odoo>` tags are mandatory anyway, so  they are considered by some as the partially-invisible-but-necessary parts of templates. In fact, [standard templates are like that](https://github.com/OCA/maintainer-tools/blob/master/template/module/views/report_name.xml). These diffs also dificult seeing the real changes.

So, please, here and everywhere, leave indentation as it was if you are changing it just for this reason :wink: